### PR TITLE
Cicero 0.21 Templates

### PR DESCRIPTION
**Pull request by jeromesimeon**
### Changes
- Upgrade all templates for the new version of Cicero (`0.21`)

### Flags
- This is now ready for review
- This does not rebuild archives (it does happen at deployment time though) mostly to make the changes easier to review
- Deployment, including the new compiled archives seems to have worked

### CODE REVIEW

File: src/copyright-license/text/sample.md [hunk 1]
```diff
@@ -14,7 +14,9 @@ Permissions. Licensor has obtained from all persons and entities who are, or who
 
 Copyright Notices. Licensee shall ensure that its use of the Work is marked with the appropriate copyright notices specified by Licensor in a reasonably prominent position in the order and manner provided by Licensor. Licensee shall abide by the copyright laws and what are considered to be sound practices for copyright notice provisions in the Territory. Licensee shall not use any copyright notices that conflict with, confuse, or negate the notices Licensor provides and requires hereunder.
 
+{{#clause paymentClause}}
```
Comment (irmerk) [inline src/copyright-license/text/sample.md:17]: The tags are in the `sample.md`?

File: src/fixed-interests/text/sample.md [hunk 2]
```diff
@@ -3,4 +3,4 @@
 This is a _fixed interest_ loan to the amount of £100,000.00
 at the yearly interest rate of 2.5%
 with a loan term of 15,
-and monthly payments of {{I'm not sure which amount right now}}
+and monthly payments of {{%I'm not sure which amount right now%}}
```
Comment (irmerk) [inline src/fixed-interests/text/sample.md:6]: Does this have a calculated value?

Comment (jeromesimeon) [reply src/fixed-interests/text/sample.md:6]: It always did. The syntax has changed.

File: src/copyright-license/text/sample.md (hunk 1 above)
Comment (jeromesimeon) [reply src/copyright-license/text/sample.md:17]: Yes. This is the same syntax as the one used in the editor (`markdown_cicero`).

**Comment by jeromesimeon** [comment]
@irmerk Thanks for the review

**Pull request merged**